src/components/table.js

```diff
@@ -726,7 +726,8 @@ export const GridTable = forwardRefWithName(
                   // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/cell_role
                   return _.flow(
                     _.groupBy('rowIndex'),
-                    Utils.toIndexPairs,
+                    _.toPairs,
+                    _.map(([k, v]) => [+k, v]),
```

Comment:
`toIndexPairs` was called with an object instead of an array here. It happened to work because the keys of this object are numeric strings, but generally that's not a valid assumption so `toIndexPairs` is typed to expect a list.